xcode/Subconscious/Shared/Components/FirstRun/FirstRunView.swift

```diff
@@ -124,6 +163,13 @@ struct FirstRunView: View {
                     FirstRunDoneView(app: app)
                 }
             }
+            .onAppear {
+                guard app.state.sphereIdentity == nil else {
+                    return
+                }
+                
+                app.send(.createSphere)
+            }
```

Comment:
Create the sphere earlier so we can kick off provisioning as soon as the invite code is accepted.

Reply:
If proceeding without an accepted invite is an actual logical failure case, we may need to add a guard to the update function as well, ensuring an invite code has been accepted before pushing another panel onto the stack. The update function should be self-standing, in the sense that it should not be able to produce an invalid next state from an action.

The UI is blocking the production of this action, and the user should never actually encounter this case, we could just guard, no-op, and log.

Reply:
Yep, agreed, right now this is driven by NavigationLinks but we need a `.pushStep` action for the FirstRun experience.